src/go/rdctl/cmd/shell.go

```diff
@@ -99,3 +104,24 @@ func setupLimaHome() error {
 	os.Setenv("LIMA_HOME", candidatePath)
 	return nil
 }
+
+func checkLimaIsRunning(commandName string) bool {
+	const howToStartMessage = "Either run `rdctl start` or start the Rancher Desktop application."
+	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	if err == nil {
+		if strings.HasPrefix(string(output), "Running") {
+			return true
+		} else {
+			fmt.Fprintf(os.Stderr, fmt.Sprintf(
+				`The Rancher Desktop VM status is currently "%s",
+but needs to be "Running" to shell into it.
+%s
```

Comment:
The line wrapping here seems inconsistent with `howToStartMessage`.

Reply:
Also, "…to shell into it" seems fine for colloquial use but seems awkwardly phrased for documentation. Consider "… needs to be Running to execute rdctl shell". (Alternatively, "rdctl shell requires Rancher Desktop to be Running; however, it is currently %s.")

Reply:
Thanks, that works better.